index.js

```diff
@@ -77,6 +78,27 @@ function getMainArgs() {
   return ArrayPrototypeSlice(process.argv, 2);
 }
 
+/**
+ * In strict mode, throw for possible usage errors like --foo --bar
+ *
+ * @param {string} longOption - long option name e.g. 'foo'
+ * @param {string|undefined} optionValue - value from user args
+ * @param {string} shortOrLong - option used, with dashes e.g. `-l` or `--long`
+ * @param {boolean} strict - show errors, from parseArgs({ strict })
+ */
+function checkOptionLikeValue(longOption, optionValue, shortOrLong, strict) {
+  if (strict && isOptionLikeValue(optionValue)) {
+    // Only show short example if user used short option.
+    const example = (shortOrLong.length === 2) ?
+      `'--${longOption}=-XYZ' or '${shortOrLong}-XYZ'` :
```

Comment:
rather than `-XYZ`, any reason not to make this `--${longOption}=${optionValue}`.

Reply:
Short version: Quoting.

I actually had that but then worried about quoting it correctly in all cases, on all platforms, so it would work if copied and pasted. Went for a short safe placeholder!

Reply:
(I'll have a look for a future PR at using the value in the error message if it is plain text.)